src/partisan_client_server_peer_service_manager.erl

```diff
@@ -401,15 +413,18 @@ establish_connections(Pending, Membership, Connections) ->
 %% keys in the dict pointing to undefined if they are disconnected or a
 %% socket pid if they are connected.
 %%
-maybe_connect({Name, _, _} = Node, Connections0) ->
-    Connections = case dict:find(Name, Connections0) of
+-spec maybe_connect(Node :: node_spec(),
+                    Connections :: partisan_peer_service_connections:t()) ->
+            partisan_peer_service_connections:t().
+maybe_connect(Node, Connections0) ->
```

Comment:
I'm not so sure about this one either.

Ideally, a single node name could have multiple connections, so I think the dictionary still needs to be indexed by the short name.  For instance, if node lasp@127.0.0.1 has 3 open connections.

We don't support via the API multiple connections yet, but that's work that's happening concurrently.  Can we adapt this accordingly (or rather, to accommodate?)